### DISCUSSION TITLE
市场痛点：链上预言机价格，急需降低成本

### DISCUSSION THREAD

**Discussion by zhangtaotaoBTC** [rebase-network/hello-world#70] [Ideas]
最近看到Discord里面一个有意思的讨论，一个用户问Argent钱包能否整合MVI和Game指数代币(由Darkforest Capital 设计的一篮子指数代币)

某负责人说，他暂时没钱来支持。因为新增一个Price Oracle需要 $1K/每月费用。他补充DPI代币的价格预言机费用是1300$每月。

因为我不是搞技术的，想问问各位，链上预言机Chainlink费用这么高吗？

市场痛点有了
市场机会：如果我们国人，搞个去中心化预言机，把费用收低一些，杀入这块市场，是否可行？
![pain point](https://user-images.githubusercontent.com/68153935/158130393-5a03bd48-b151-4169-8827-03ffa8ff7f69.JPG)

**Comment by zhangtaotaoBTC** [comment]
@wartstone @luckyyang 老哥，你们怎么看？

**Reply by luckyyang** [reply]
预言机是有强需求的，我不知道你要是搞能不能成功，不过作为用户，我肯定希望你搞出来一个便宜又好用的 oracle 的
